### PR TITLE
Adds a multiply function and tests

### PR DESCRIPTION
**Pull request by php23twb**

### CODE REVIEW

**Comment by php23twb** [comment]
closes #4

File: pythoncalculator.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```
Comment (abhinovenagarajans) [inline pythoncalculator.py]: I think this needs to be changed to 
```suggestion
    return x*y

```

**Review by abhinovenagarajans — CHANGES_REQUESTED** [review]

**Review by abhinovenagarajans — APPROVED** [review]
Yes, this looks good @php23twb .

**Review by Mimz-i — CHANGES_REQUESTED** [review]
No module named 'pythoncalculator.multiply'.
pythoncalculator/pythoncalculator.py needs renaming with above.

**Review by abhinovenagarajans — APPROVED** [review]
Changes look good @php23twb

**Pull request merged**